OmniFocus 3 & 4/Statistics/OF Statistics.omnijs

```diff
@@ -61,7 +61,8 @@
                     return compose(
                         map(second(length)),
                         groupSort,
-                        map(tupleFromObj)
+                        map(tupleFromObj),
+                        filter(o => !(o.constructor.name === 'Task' && o.project)) // exclude tasks which are a root task of a project
```

Comment:
OmniFocus `Task.project` returns the `Project` that the task is the root task of, or `null`

Reply:
I am glad you like my Plug-In. Do explain to me what is your logic there:

`!(o.constructor.name === 'Task' && o.project)`

not (Object `o`  belongs to Task class and `o` containing project is not null ?

Reply:
To avoid including projects in the count of tasks, we need to exclude `task` objects which are _root_ tasks of projects (ie. which correspond to the title, status, etc of the project).

The `Task.project` property is `true` for these types of tasks. Note: this property does not return the _parent project_ of the task, that is given by `Task.containingProject → Project`.

I guess this could be simplified to `!o.project` since that property returns `undefined` for all objects which are not tasks. The constructor check was just a bit of redundant safety.